wasm64 suite `--profiling-funcs` regression

It looks like PR https://github.com/emscripten-core/emscripten/pull/26963/changes slipped in uses of `--profiling-funcs`  to the wasm64 suite.

But at the same time, the test suites are run with `-Werror`, so this results in a number of tests failing with

```
C:\emsdk\emscripten\main>test\runner wasm64.test_dylink_dot_a
Running 1 tests
[1/1] test_dylink_dot_a (test_core.wasm64.test_dylink_dot_a) ... FAIL

Stdout:

emcc: error: linker flag ignored during compilation: '--profiling-funcs' [-Wunused-command-line-argument] [-Werror]



======================================================================
FAIL: test_dylink_dot_a (test_core.wasm64.test_dylink_dot_a)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\emsdk\emscripten\main\test\common.py", line 1211, in run_process
    rtn = utils.run_process(cmd, check=check, **kwargs)
  File "C:\emsdk\emscripten\main\tools\utils.py", line 45, in run_process
    ret = subprocess.run(cmd, check=check, input=input, *args, **kw)
  File "C:\emsdk\python\3.13.3_64bit\Lib\subprocess.py", line 577, in run
    raise CalledProcessError(retcode, process.args,
                             output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['C:\\emsdk\\emscripten\\main\\emcc.bat', '-fPIC', '-c', 'third.c', '-o', 'third.o', '-Wclosure', '-Werror', '-Wno-limited-postlink-optimizations', '--profiling-funcs', '-m64']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\emsdk\emscripten\main\test\common.py", line 380, in resulting_test
    return func(self, *args)
  File "C:\emsdk\emscripten\main\test\test_core.py", line 235, in decorated
    return func(self, *args, **kwargs)
  File "C:\emsdk\emscripten\main\test\test_core.py", line 5086, in test_dylink_dot_a
    self.run_process([EMCC, '-fPIC', '-c', 'third.c', '-o', 'third.o'] + self.get_cflags(compile_only=True))
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\emsdk\emscripten\main\test\common.py", line 1216, in run_process
    self.fail(f'subprocess exited with non-zero return code({e.returncode}): `{shlex.join(cmd)}`')
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: subprocess exited with non-zero return code(1): `'C:\emsdk\emscripten\main\emcc.bat' -fPIC -c third.c -o third.o -Wclosure -Werror -Wno-limited-postlink-optimizations --profiling-funcs -m64`

Stdout:

emcc: error: linker flag ignored during compilation: '--profiling-funcs' [-Wunused-command-line-argument] [-Werror]


----------------------------------------------------------------------
Ran 1 test in 0.335s

FAILED (failures=1)
```

See e.g. http://clbri.com:8010/api/v2/logs/443833/raw_inline

--- a/test/common.py
+++ b/test/common.py
@@ -842,7 +842,9 @@ class RunnerCore(RetryableTestCase, metaclass=RunnerMeta):
   #                  libraries, for example
   def get_cflags(self, main_file=False, compile_only=False, asm_only=False):
     def is_ldflag(f):
-      return f.startswith('-l') or any(f.startswith(s) for s in ['-sEXPORT_ES6', '-sGL_TESTING', '-sPROXY_TO_PTHREAD', '-sENVIRONMENT=', '--pre-js=', '--post-js=', '-sPTHREAD_POOL_SIZE='])
+      return f.startswith(('-l', '-sEXPORT_ES6', '-sGL_TESTING', '-sPROXY_TO_PTHREAD',
+                           '-sENVIRONMENT=', '--pre-js=', '--post-js=', '-sPTHREAD_POOL_SIZE=',
+                           '--profiling-funcs'))
 
     args = self.serialize_settings(compile_only or asm_only) + self.cflags
     if asm_only:

